src/pages/BonusQuestion.tsx

```diff
@@ -1,34 +1,90 @@
 import { IonButton } from '@ionic/react'
-import React from 'react'
+import React, { useMemo, useEffect } from 'react'
+import { insert, sortBy } from 'ramda'
 import { useHistory } from 'react-router'
 
 import StatsLayout from '../components/layouts/StatsLayout'
+import { useQuery, gql, useLazyQuery } from '@apollo/client'
+import { MovieData, MOVIES } from './Question'
+import MovieCard from '../components/card/MovieCard'
 import useStore from '../useStore'
 
+const GET_BONUS_SCORE = gql`
+  query($imdbIds: [String!]!, $titleQuestionScores: Int!) {
+    scoreBonusResponse(
+      imdbIds: $imdbIds
+      titleQuestionScores: $titleQuestionScores
+    )
+  }
+`
+
 const BonusQuestion = () => {
   const history = useHistory()
 
-  const { setPointDifference } = useStore()
+  const { setPointDifference, pointDifference } = useStore()
+
+  const { loading, data } = useQuery<MovieData>(MOVIES, {
+    fetchPolicy: 'cache-only',
```

Comment:
Use the same random movies as in the initial question. We could also put
```suggestion
    fetchPolicy: 'network-only',
```
if we want three different random movies.